src/main/resources/db/migration/V1_8__add_outcome_status_to_visit_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE visit ADD outcome_status VARCHAR(40);
```

Comment:
Are we no longer dropping the schemas?

Reply:
I thought this is now in pre-prod?

Reply:
Yeah but just to test the deployment changes - im not aware of anyone actually using it yet - will check in stand-up

Reply:
Yes but do we have access to DB to drop it and start again?

Reply:
Yeah we do - new port forward pod pointing to preprod instead of dev